lib/charliecloud.py

```diff
@@ -1155,16 +1155,13 @@ def root_default():
 
    @staticmethod
    def root_env():
+      if ("CH_GROW_STORAGE" in os.environ):
+         # Avoid surprises if user still has $CH_GROW_STORAGE set (see #906).
+         FATAL("$CH_GROW_STORAGE no longer supported; use $CH_IMAGE_STORAGE")
```

Comment:
Do we want an issue to remove this eventually? FWIW, I don't see any harm in it staying.

Reply:
I was going to leave it. Eventually it will become obsolete, when the last person using `$CH_GROW_STORAGE` stops. However I don't know how to figure out when that happens. I made deferred issue #1007.